### DISCUSSION TITLE
I got some questions regarding Premium Video-Download App for Firefox.

### DISCUSSION THREAD

**Discussion by Assassin7O7980** [aclap-dev/video-downloadhelper#1535] [Help and Q&A]
### Your question

Hello there! I got some questions that hopefully I remember so...

My System is Windows 11 and I am planning on using this app on my Mozilla Firefox Internet Browser.

Umm... For Premium... Is it possible to pay in Paypal any how?... Or not really?.. Also I been wondering... How or where would the license key be attached? Would it be attached to my Firefox Account or something else entirely?..

Thank you! That's all for now... Please take care and I hope to see your reply as soon when I can.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 1
}

**Comment by RamenAlchemist** [comment]
I'm going to say don't bother paying for this. I Paid for this maybe an hour or two ago, tried to download some videos, and the videos defaulted to a single audio track in spanish, despite it being a mkv. I looked around and i'm still trying to figure out how to get my videos to come out with english audio. There have been some very helpful posts in discussions but I am finding them very hard to follow. So i would say, buy at your own risk.

**Comment by Mattmjs** [comment]
@Assassin7O7980 , Paypal is not supported, a credit or debit card are your options. The license is added in the settings page in the
extension.
